vyper/ir/compile_ir.py

```diff
@@ -555,13 +556,10 @@ def _height_of(witharg):
         o = _compile_to_assembly(code.args[0], withargs, existing_labels, break_dest, height)
         o.extend(
             [
-                "PUSH1",
-                MemoryPositions.FREE_VAR_SPACE,
```

Comment:
Is this only 0 or 1?

Reply:
right now FREE_VAR_SPACE is 0, but since it could change, use the PUSH abstraction